gptables/core/gptable.py

```diff
@@ -172,7 +167,7 @@ def _valid_column_index(self, column_index):
         return column_index in range(self.table.shape[1])
         
 
-    def _set_column_headings(self):
+    def _set_column_headings(self): # TODO: check custom formatting in headers
```

Comment:
if this means using your own theme to change table headers format then this works fine.

Reply:
That's great that the theme is still working correctly. The issue is if people want to apply different formatting to specific words in the column headings. There is a WIP branch looking at this issue: [units-custom-formatting](https://github.com/best-practice-and-impact/gptables/tree/units-custom-formatting)